### DISCUSSION TITLE
User reports (preferably do them in the discord server)

### DISCUSSION THREAD

**Discussion by JonathanHowie** [red-stone-network/clockwork#17] [General]
Report people here.

``TO-DO: move to muenster repo - lukasexists``

EDIT FROM LUKASEXISTS: This is obsolete now! Contact support at support@mail.redstonenetwork.rit.cl . For more important matters contact Trust and Safety at trustsafety@mail.redstonenetwork.rit.cl . And for DMCA takedown requests, contact us at dmca-takedowns@mail.redstonenetwork.rit.cl .

**Comment by JonathanHowie** [comment]
![image](https://user-images.githubusercontent.com/58430223/220383792-53e960ad-708b-45a9-9c8a-dbcaddc8d3d5.png)
![image](https://user-images.githubusercontent.com/58430223/220383810-5cc1ee43-7573-4978-adec-0badcb78e3d3.png)
![image](https://user-images.githubusercontent.com/58430223/220383821-43637a17-60bb-4827-bb55-6e89fe91c9f1.png)

**Reply by ToasterPanic** [reply]
for future reference dont claim to be an admin, thats against rules

**Comment by ToasterPanic** [comment]
you're supposed to report in the discord, ill handle him in a bit. rn glitch is blocked for me so

**Comment by ToasterPanic** [comment]
situations been handled
